projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -1,75 +1,66 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import logging
+
 from airflow.providers.http.hooks.http import HttpHook
-from tenacity import sleep
-from tenacity import stop_after_attempt
-from vdk.internal.control.auth.auth import Authentication
+from taurus_datajob_api import DataJobExecutionRequest
+from vdk.internal.control.rest_lib.factory import ApiClientFactory
```

Comment:
I believe we agreed there's no value in re-using ApiClientFactory since most of it is configuration and in the provider the configuration is provided in another way (connection attributes) than one here.

Reply:
dropped ApiClientFactory and included all config inside the hook class